src/themes/schulportal/login/template.ftl

```diff
@@ -100,45 +100,54 @@
                             <div class="col-md-10">
                                 <#nested "show-username">
                                 <div id="kc-username" class="${properties.kcFormGroupClass!}">
-                                    <label id="kc-attempted-username">${auth.attemptedUsername}</label>
-                                    <a id="reset-login" href="${url.loginRestartFlowUrl}" aria-label="${msg("restartLoginTooltip")}">
-                                        <div class="kc-login-tooltip">
-                                            <i class="${properties.kcResetFlowIcon!}"></i>
-                                            <span class="kc-tooltip-text">${msg("restartLoginTooltip")}</span>
-                                        </div>
-                                    </a>
+                                    <label id="kc-attempted-username">${auth.attemptedUsername}</label>                                    
                                 </div>
                             </div>
                         </div>
                     <#else>
                         <#nested "show-username">
                         <div id="kc-username" class="${properties.kcFormGroupClass!}">
-                            <label id="kc-attempted-username">${auth.attemptedUsername}</label>
-                            <a id="reset-login" href="${url.loginRestartFlowUrl}" aria-label="${msg("restartLoginTooltip")}">
-                                <div class="kc-login-tooltip">
-                                    <i class="${properties.kcResetFlowIcon!}"></i>
-                                    <span class="kc-tooltip-text">${msg("restartLoginTooltip")}</span>
-                                </div>
-                            </a>
+                            <label id="kc-attempted-username">${auth.attemptedUsername}</label>                            
                         </div>
                     </#if>
                 </#if>
             </header>
+
+            <#-- SPSH: custom messages for failed OTP authentication -->
+            <#-- Remove all whitespaces and linebreaks from the message key to-->
+            <#assign customMessages = {
+                "Authenticationfailed.falscherOTP-Wert": msg("authentifactionOtpFailedMessage"),
+                "Authenticationfailed.falscherOTP-Pin": msg("authentifactionOtpFailedMessage"),
+                "Authenticationfailed.wrongotpvalue": msg("authentifactionOtpFailedMessage"),
+                "Authenticationfailed.wrongotppin": msg("authentifactionOtpFailedMessage")
```

Comment:
Maybe my reading-comprehension is failing me on a friday, but I feel like `Authenticationfailed` is not a word

Reply:
Nope, definitely my reading-comprehension.

Reply:
you're right, there was a spelling mistake. fixed to authenticationOtpFailedMessage